### PR TITLE
IGAPP-835: Remove enzyme on web

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Removed tests which only contained snapshot-testing. I can open a ticket with a list of these files, so we write new test for them, if this is wanted. Removed snapshots.

We can also add a custom render-function, which applies the theme automatically, as we have a lot of tests which manually add the ThemeProvider. I can open a ticket for this, too, if wanted.

### CODE REVIEW

File: web/src/components/FeedbackToolbarItem.tsx [hunk 1]
```diff
@@ -36,7 +36,11 @@ const FeedbackToolbarItem = ({
   return (
     <Tooltip text={viewportSmall ? null : dataTip} flow='up' mediumViewportFlow='right' smallViewportFlow='down'>
       <StyledFeedbackToolbarItem className={className} onClick={handleLinkClick} aria-label={dataTip}>
-        <FontAwesomeIcon className={className} icon={isPositiveRatingLink ? faSmile : faFrown} />
+        <FontAwesomeIcon
+          className={className}
+          icon={isPositiveRatingLink ? faSmile : faFrown}
+          title={isPositiveRatingLink ? 'faSmile' : 'faFrown'}
```
Comment (steffenkleinle) [inline web/src/components/FeedbackToolbarItem.tsx:42]: Title should not be used for this purpose, it is shown as a tooltip:
image
I think you could just remove the tests, they don't really test important stuff.

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
Finally no more enzyme :tada:

**Comment by steffenkleinle** [comment]
> Removed tests which only contained snapshot-testing. I can open a ticket with a list of these files, so we write new test for them, if this is wanted. Removed snapshots.

Wouldn't do this tbh, those issues just live in the backlog for years until they are rejected. When new behavior is added or changed, we should write tests.

> We can also add a custom render-function, which applies the theme automatically, as we have a lot of tests which manually add the ThemeProvider. I can open a ticket for this, too, if wanted.

There are already custom render functions but with `wrapWithTheme` set to false, but you are right, no "basic" render function yet. This could just be changed to fix the problem. However, it is now recommended to use the MemoryRouter if react-router functionality is used. Maybe we should open a ticket to change that.

File: web/src/components/__tests__/NewsTabs.spec.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+import { renderWithBrowserRouter } from '../../testing/render'
 import NewsTabs from '../NewsTabs'
 
 describe('NewsTabs', () => {
   const language = 'en'
   const t = (key: string) => key
 
   it('should render two tabs if both local news and tunews are enabled', () => {
-    const wrapper = shallow(
+    const { getByLabelText } = renderWithBrowserRouter(
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply web/src/components/__tests__/NewsTabs.spec.tsx:14]: Tried everywhere without and test failed, because of href tags.

**Comment by f1sh1918** [comment]
please resolve merge conflicts :)

**Review by steffenkleinle — APPROVED** [review]

**Comment by SteffiStuffel** [comment]
> please resolve merge conflicts :)

That's not our workflow, because we would have a lot of commits and overhead, if we always keep branches conflict free, while waiting for reviews.

Moreover in this case. these conflicts are just trailing commas stuff.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**